app.rb

```diff
@@ -107,8 +107,17 @@ def home_page
   def saml_settings
     template = File.read('config/saml_settings.yml')
     base_config = Hashie::Mash.new(YAML.safe_load(ERB.new(template).result(binding)))
+
+    # TODO: don't use the demo cert and key in EC2 environments
+    if LoginGov::Hostdata.in_datacenter? && (
+      LoginGov::Hostdata.domain == 'login.gov' ||
+      LoginGov::Hostdata.env == 'prod'
+    )
+      raise NotImplementedError.new('Refusing to use demo cert in production')
```

Comment:
This looks like it prevents the app from running at all in prod. Presumably that's okay since this app shouldn't be deployed in prod.

Reply:
Yeah that's the idea, to prevent use of the publicly disclosed demo key in prod.